pkg/graph/resolvers/model_plan.resolvers.go

```diff
@@ -109,9 +110,47 @@ func (r *modelPlanResolver) Tdls(ctx context.Context, obj *models.ModelPlan) ([]
 
 // EchimpCRsAndTDLs is the resolver for the echimpCRsAndTDLs field.
 func (r *modelPlanResolver) EchimpCRsAndTDLs(ctx context.Context, obj *models.ModelPlan) ([]models.EChimpCRAndTDLS, error) {
+	// TODO Update to use flag value to conditionally use SQL/DB calls instead of S3 ECHIMP Cache
```

Comment:
I think this TODO can probably be removed.

Reply:
Addressed in 84f4ed993